### DISCUSSION TITLE
How do you handle name casing?

### DISCUSSION THREAD

**Discussion by kuanyinglu** [helix-editor/helix#5352] [General]
Just wondering how people have dealing with variable names with camel casing, snake casing, or other types?
For example ReallyLongVariableName, really_long_variable_name, REALLY_LONG_VARIABLE_NAME, or even really-long-variable-name.

Tree sitter probably won't be helpful in this case since it's not really syntax related. Normal motions like w,e,b are too big for camel casing.
How do you handle it either in Vim or other editor? Textobject? Inventing smaller motions? Search for character? Delete the word and retype everything?

**Comment by dead10ck** [comment]
Really not sure what you're asking. What do you mean by "handle"? What is the actual problem you are having?

**Reply by kuanyinglu** [reply]
It's not really a problem, but a question on how people efficiently edit texts like these. For example, if I have an function like GetCustomerLastName, then I copy and paste to start working on a new endpoint, let's say GetEmployeeFirstAndLastName, now I have to rename it, and possibly a bunch of variables names inside the function as well. I'm just wondering how people have been editing them fast and efficiently.
